test/run-tests

```diff
@@ -100,6 +100,7 @@ def handle_transient_httperrors(error):
             transient_error_wait_time.next(),
         )
 
+        logging.info("Rate limiting hit, waiting for %s seconds", int(timeout))
         print(f">>> Rate limiting hit, waiting for {int(timeout)} seconds...")
```

Comment:
Since this is basically the same message, how about adding a second log handler for level warn that outputs to stdout so that there is not need for a logging message and a print?

Reply:
OK, I'll have to rework `setup_logging()` though